apm-sniffer/apm-sdk-plugin/mssql-commons/src/main/java/org/apache/skywalking/apm/plugin/mssql/commons/PreparedStatementExecuteMethodsInterceptor.java

```diff
@@ -47,6 +49,13 @@ public final void beforeMethod(EnhancedInstance objInst, Method method, Object[]
             Tags.DB_INSTANCE.set(span, connectInfo.getDatabaseName());
             Tags.DB_STATEMENT.set(span, SqlBodyUtil.limitSqlBodySize(cacheObject.getSql()));
             span.setComponent(connectInfo.getComponent());
+            if (JDBCPluginConfig.Plugin.JDBC.TRACE_SQL_PARAMETERS) {
```

Comment:
TRACE_SQL_PARAMETERS is false in default, you need to activate it in the test scenario. I can't see the change, could you recheck?

Reply:
Maybe I can try. This is the test picture.
![2](https://user-images.githubusercontent.com/19707438/124063299-1a233880-da65-11eb-85b0-e9350b049c6a.png)
![1](https://user-images.githubusercontent.com/19707438/124063256-04ae0e80-da65-11eb-982d-7dc90dd4f205.jpg)